src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -507,5 +507,16 @@ public static int countEmptyStrings(String... strings) {
      */
     public static String convertToEmptyStringIfNull(String str) {
         return (str == null) ? "" : str;
-    }  
+    }
+    
+    /**
+     * Removes both the (square) brackets surrounding a string array.
+     * Brackets are assumed to be the first and last character of a string array.
+     * 
+     * @param arrayString
+     * @return the string without being enclosed with the square brackets 
+     */
+    public static String removeBracketsForArrayString(String arrayString) {
+        return arrayString.substring(1, arrayString.length() - 1).trim();
+    }
```

Comment:
Now that this is a general utility method, make it look like a general utility method, not a method specific to a certain usage. e.g. it can be named `removeEnclosingBrackets`. The implementation needs to be defensive (e.g. check if the string has brackets to begin with) and unit tested.